exporter/datadogexporter/metrics_exporter.go

```diff
@@ -99,7 +99,7 @@ func translatorFromConfig(logger *zap.Logger, cfg *Config, sourceProvider source
 }
 
 func newMetricsExporter(ctx context.Context, params exporter.CreateSettings, cfg *Config, onceMetadata *sync.Once, sourceProvider source.Provider, apmStatsProcessor api.StatsProcessor) (*metricsExporter, error) {
-	client := clientutil.CreateZorkianClient(cfg.API.Key, cfg.Metrics.TCPAddr.Endpoint)
+	client := clientutil.CreateZorkianClient(string(cfg.API.Key), cfg.Metrics.TCPAddr.Endpoint)
```

Comment:
Perhaps we should change the type of this parameter, and then case once in the constructor, rather than casting in so many places in callers?

Reply:
I didn't want to bother on the Zorkian bits since we will remove them soon-ish, but it's a good idea. @songy23 what do you think?

Reply:
Agreed, Zorkian is going away soon so I'd vote to not bother doing too much refactoring.